Polyfills/AbortController/Readme.md

```diff
@@ -0,0 +1,8 @@
+# AbortController
+Implements parts of [`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController/). Provides a way to trigger the abort signal. *Work In Progress*

```

Comment:
Probably should put both AbortController and AbortSignal here.

Reply:
Oh I see what you did here. Can you please collapse both classes into one folder and one polyfill? Just put the AbortSignal.* in the AbortController folder and CMakeLists.

Reply:
Call the polyfill ABORT_CONTROLLER and drop the SIGNAL.